lib/elastic_apm/util.rb

```diff
@@ -46,6 +46,8 @@ def self.reverse_merge!(first, *others)
 
     def self.truncate(value, max_length: 1024)
       return unless value
+
+      value = String(value)
```

Comment:
Just curious why this change is included- was there a bug with this?

Reply:
Yes, the `account.id` was an integer on, if I remember correctly, Google Cloud Engine.